django_geosource/models.py

```diff
@@ -177,9 +177,9 @@ def update_fields(self):
                 if field_name not in fields:
                     field, is_new = self.fields.get_or_create(
                         name=field_name,
-                        defaults={"label": field_name},
-                        order=i,
+                        defaults={"label": field_name, "order": i},
                     )
+                    field.order = i  # force order for update
```

Comment:
https://docs.djangoproject.com/fr/3.1/ref/models/querysets/#update-or-create ?

Reply:
No, we don't want to set label on update.